examples/sst2/train.py

```diff
@@ -154,7 +154,7 @@ def loss_fn(model):
     loss = loss + l2_penalty
     return loss, logits
 
-  loss, _, grad = optimizer.compute_gradient(loss_fn)
+  loss, _, grad = jax.value_and_grad(loss_fn, has_aux=True)(optimizer.target)
```

Comment:
It's really unfortunate that this example lacks tests. I don't think this change is correct. Shouldn't it be `(loss, _), grad = [...]`?

(Could you please run this example to make sure? Or if you're really inspired maybe file a separate pull request just making sure that this example can run a single training step without failing, with small hyperparameters? See #176 )

Reply:
You're right, going to fix it. Also, this example won't run because flax.training is not exposed, I changed that too

Reply:
One epoch for this guy as well:

```
I0422 20:08:32.119502 139793491572544 train.py:233] Epoch 01 train loss 0.4759 valid loss 0.4421 acc 81.19
```

Reply:
Thanks for testing @danielsuo!